admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+= Aggregating Logs with the EFK Stack
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Application developers can aggregate and view logs for a range of services using
```

Comment:
This is a cluster admin task, but the intro makes it sound like "application developers" (OpenShift users) can set all this up. I think the intro should make the distinction more clear, and also give more context on what "a range of services"  means, and why a developer would be interested.